### ISSUE TITLE
Suggestion: build a DB Explorer UI app - like PHPMyAdmin for MySQL

### ISSUE THREAD

**Issue by zoltan-fedor** [semi-technologies/weaviate#1946]
While this might seems to be not important at first, I do believe that it would be very beneficial to build a DB explorer tool, like PHPMyAdmin for MySQL or at least support integration with DB Editors, like DBeaver.

This could be very useful in operations and troubleshooting when the Weaviate DB is used for multiple things and stores a high number of documents.
While writing queries manually is always possible, having a click-able explorer tool would save a lot of time.

Basically this could be just an addition to the Weaviate Console allowing browsing of classes and documents.

Thanks
